TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlay/PowerPlayAutoRoadRunner.java

```diff
@@ -173,7 +169,8 @@ public void onError(int errorCode) {
         timer.start(); // Start match timer as soon as start
         cam.stopStreaming();
         cam.closeCameraDevice();
-        trajectoryTwo = buildTrajectoryTwo(side, foundTag, drive);
+//        trajectoryTwo = buildTrajectoryTwo(side, foundTag, drive);
+        trajectoryTwo = trajectoryCollection.getTrajectory(side, foundTag).getParkTrajectory();
         if(side == InitialSide.RIGHT) {
             trajectoryNoScoring = buildTrajectoryNoScoring(side, drive, START_POS_RIGHT, foundTag);
```

Comment:
Suggest adding trajectoryNoScoring to the TrajectoryCollection.  But actually, you know from Options Op whether you need NoScoring or One and Two, so you don't have to generate all of them at all.